CHANGELOG.md

```diff
@@ -37,6 +37,7 @@ All notable changes to this project will be documented in this file.
 * ### NI-SCOPE
   * #### Added
   * #### Changed
+    * Added default values for fetch and read
```

Comment:
Please add the full list of functions you added default values to.

Reply:
Fixed